algosdk/atomic_transaction_composer.py

```diff
@@ -33,6 +34,38 @@ class AtomicTransactionComposerStatus(IntEnum):
     COMMITTED = 4
 
 
+def populate_foreign_array(
+    value_to_add: Any, foreign_array: List[Any], zero_value: Any = None
```

Comment:
nit: you could use generics here: https://mypy.readthedocs.io/en/stable/generics.html#generic-functions